cmd/cli/info.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+
+	"github.com/spf13/cobra"
+)
+
+func isURL(arg string) bool {
+	_, err := url.ParseRequestURI(arg)
+	return err == nil
+}
+
+var infoCmd = &cobra.Command{
+	Use:   "info",
+	Short: "Provides informations about a puglin",
+	Long:  "Retrieves the .reef.yml from the repository and displays the informations.",
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 || !isURL(args[0]) {
```

Comment:
You could use directly the `_, err := url.ParseRequestURI(arg)` here and return the `error`. Using the `isURL` function you lose the error message